01-messaging.md

```diff
@@ -124,7 +124,9 @@ errors which make channels unusable or further communication unusable.
 A node MAY send an empty [data] field.  A node sending `error` MUST
 fail the channel referred to by the `channel-id`, or if `channel-id`
 is `0xFFFFFFFFFFFFFFFF` it MUST fail all channels and MUST close the connection.
-A node MUST NOT set `len` to greater than the data length.
+A node MUST NOT set `len` to greater than the data length.  A node
+SHOULD include the raw, hex-encoded transaction in the case where failure
+was caused by an invalid signature check.
```

Comment:
Maybe:
"A node SHOULD include the raw, hex-encoded transaction in reply to a `funding_created`, `funding_signed`, `closing_signed` or `commitsig` message when failure was caused by an invalid signature check"?

btw, could we rename `commitsig` -> `commit_signed` or `commitment_signed`? I know, I am obsessed with consistency 😸